tests/security/authz_jwt/authz_jwt_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authzjwt
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+
+	"istio.io/istio.io/pkg/test/istioio"
+)
+
+//https://istio.io/docs/tasks/security/authorization/authz-jwt/
+//https://github.com/istio/istio.io/blob/release-1.5/content/en/docs/tasks/security/authorization/authz-jwt/index.md
+func TestAuthzJWT(t *testing.T) {
+	framework.
+		NewTest(t).
+		Run(istioio.NewBuilder("tasks__security___authz_jwt").
+			Add(istioio.Script{
+				Input: istioio.Inline{
+					FileName: "create_ns_foo_with_httpbin_sleep.sh",
+					Value: `
+source ${REPO_ROOT}/content/en/docs/tasks/security/authorization/authz-jwt/snips.sh
+snip_before_you_begin_1`,
+				},
+			}).
+			Add(istioio.MultiPodWait("foo")).
```

Comment:
Just FYI, part of the cleanup exercise I'm doing is to try to make all tests put the entire test in one .sh file (excluding cleanup which needs to be deferred, although I have an idea for handling that too, to be tried in a future cleanup). Rather than relying on code in this go file to wait for pods to deploy, I'd like the script to just call a bash library function to do it, or similar wait. Once all the tests are completely described in one bash file, this go file becomes total boilerplate, so we should be able to either generate it, or even do it on-the-fly.

Reply:
That sounds like a great plan to me.  I wanted to finish knocking out the rest of the security task tests, so I'll plan to write them in the current style.  I'm happy to convert them when you're ready, though.